src/main/controllers/AllJudgmentsController.ts

```diff
@@ -0,0 +1,52 @@
+import { Response } from 'express';
+
+import { AppRequest } from '../definitions/appRequest';
+import { TranslationKeys } from '../definitions/constants';
+import { FormContent } from '../definitions/form';
+import { AnyRecord } from '../definitions/util-types';
+
+import { getPageContent } from './helpers/FormHelpers';
+import {
+  getDecisions,
+  getJudgments,
+  populateDecisionItemsWithRedirectLinksCaptionsAndStatusColors,
+  populateJudgmentItemsWithRedirectLinksCaptionsAndStatusColors,
+} from './helpers/JudgmentHelpers';
+
+export default class AllJudgmentsController {
+  public get = async (req: AppRequest, res: Response): Promise<void> => {
+    const userCase = req.session?.userCase;
+
+    let judgments = undefined;
```

Comment:
For all these, you don't need to set it explicitly as undefined
For all of these, it might also be nicer to have the ternary let x = (a) ? b : c - depends on the length of the expression I suppose, this isn't as crucial as the first point.

Reply:
Removed undefined